Lib/tkinter/test/test_tkinter/test_widgets.py

```diff
@@ -940,7 +940,9 @@ def test_digits(self):
 
     def test_from(self):
         widget = self.create()
-        self.checkFloatParam(widget, 'from', 100, 14.9, 15.1, conv=float_round)
+        # A change in Tk 8.6.10 means `from` is not rounded (bpo-41306)
```

Comment:
I am going to remove this line so https://github.com/python/cpython/pull/22405/files does not need to ber rebased